azext_iot/operations/events3/_events.py

```diff
@@ -244,56 +319,67 @@ def send_c2d_message(target, device_id, data, properties=None,
 
     msg_content = str.encode(data)
 
-    message = uamqp.Message(msg_content, properties=msg_props, application_properties=app_props)
+    message = uamqp.Message(
+        msg_content, properties=msg_props, application_properties=app_props
+    )
 
-    operation = '/messages/devicebound'
+    operation = "/messages/devicebound"
```

Comment:
is there any linter in place for double quotes vs single quotes?

Reply:
This is autoformatting with python black.